diff --git a/include/libfdt_env.h b/include/libfdt_env.h
index 3667d4c..1455f1d 100644
--- a/include/libfdt_env.h
+++ b/include/libfdt_env.h
@@ -21,6 +21,18 @@
 
 #include "qemu/bswap.h"
 
+#ifdef __CHECKER__
+#define __force __attribute__((force))
+#define __bitwise __attribute__((bitwise))
+#else
+#define __force
+#define __bitwise
+#endif
+
+typedef uint16_t __bitwise fdt16_t;
+typedef uint32_t __bitwise fdt32_t;
+typedef uint64_t __bitwise fdt64_t;
+
 #ifdef HOST_WORDS_BIGENDIAN
 #define fdt32_to_cpu(x)  (x)
 #define cpu_to_fdt32(x)  (x)
